### DISCUSSION TITLE
.NET 8 support

### DISCUSSION THREAD

**Discussion by scosec** [microsoft/PowerPlatform-DataverseServiceClient#432] [Q&A] [answered]
Hi Everyone,

Does anyone have an idea when .NET 8 will be supported for the dataverse client?
If I have some idea I can take this into planning.

Thanks! :)

**Comment by MattB-msft** [comment, marked as answer]
It is supported now in .net 8,  The .net 6 build works fine and we actively test on .net 8.
We will likely not ship a target for .net 8 until .net 6 gets closer to retirement ( late this year. ) 

MattB

**Reply by FarmerenHE** [reply]
Do you have any updates on when an official .net 8 target package is coming?

My customers would like to update just a little bit in advance to have time to test it out, while also keeping on the officially supported version.

**Reply by scottdurow** [reply]
The package now targets .net 8 🚀 Thank you @MattB-msft 
![image](https://github.com/user-attachments/assets/e06c585c-44ae-4ee7-b138-497bdc762c91)

https://www.nuget.org/packages/Microsoft.PowerPlatform.Dataverse.Client

**Comment by fsmartini** [comment]
Heads up, .Net6.0 is no longer supported. ;)

**Reply by BetimBeja** [reply]
Is it November already?
![image](https://github.com/microsoft/PowerPlatform-DataverseServiceClient/assets/11160171/f53be95f-e63c-4f4f-8fff-a612db26fe01)

**Reply by fsmartini** [reply]
Sorry, my bad. Though about .Net7.0.. Short term.
